addons/rating/models/rating.py

```diff
@@ -118,11 +119,14 @@ def create(self, vals_list):
         for values in vals_list:
             if values.get('res_model_id') and values.get('res_id'):
                 values.update(self._find_parent_data(values))
+            values['rated_on'] = fields.Datetime.now()
         return super().create(vals_list)
 
     def write(self, values):
         if values.get('res_model_id') and values.get('res_id'):
             values.update(self._find_parent_data(values))
+        if 'rating' in values or 'feedback' in values:
```

Comment:
shouldn't we keep only `'rating' in values` ?

Reply:
We are also updating the date when a user changes the feedback to keep track of the latest rating/feedback date.